shop.py

```diff
@@ -0,0 +1,30 @@
+class Product:
+    def __init__(self, name, price):
+        self.name = name
+        self.price = price
+    
+    def get_total(self, amount):
+        return round(self.price * amount, 2) # round((self.price * 100)*amount /100, 2)
+    
+class ShoppingCart:
+    def __init__(self):
+        self.purchases = []
+        self.goods_price = []
+
+    def add(self, product, amount):
+        goods = [product for i in range(amount)]
+        self.goods_price.append(product.price*amount)
```

Comment:
Он у тебе метод вище для цього є, а тут ти про округлення забув.
```suggestion
        self.goods_price.append(product.get_total(amount))
```

Reply:
я гублюсь зберігати кількість товару чи сумму ціни товару